### PR TITLE
Fix plot scripts

### LINKED ISSUE #4316: MSL v4.1.0-beta.1 feedback: broken Dymola plot scripts
There are 3 plot scripts in MSL 4.1.0 beta 1 with errors:

1.  _plotResults_ in `Modelica.Fluid.Examples.HeatingSystem`
    `Error: Could not plot variable 'pipe.heatPort[1].Q_flow'`

2. _plot level and port.m_flow_ in `Modelica.Fluid.Examples.Tanks.ThreeTanks`
    `Error: Could not plot variable 'tank3.ports[2].m_flow'`

3. _plot level and port.m_flow_ in `Modelica.Fluid.Examples.Tanks.TanksWithOverflow`
    `Error: Could not plot variable 'tank3.ports[2].m_flow'`

Tested with Dymola Version 2024x Refresh 1 Alpha.

### PR DESCRIPTION
**Pull request by HansOlsson**
Closes #4316
I believe these make sense. I don't know exactly what they intended to show, but:
- wall and heater have somewhat similar heat-flows showing that you heat and it is lost in the wall later on
- in the other two cases there were already enough variables plotted in that diagram, so I don't think we need the extra one

### CODE REVIEW

File: Modelica/Resources/Scripts/Dymola/Fluid/TanksWithOverflow/plot level and port.m_flow.mos [hunk 1]
```diff
@@ -64,8 +64,7 @@ createPlot(id = 1,
  bottomTitleType = 1);
 createPlot(id = 1,
  position = {3, 3, 771, 270},
- y = {"upperTank.ports[1].m_flow", "pipe.port_a.m_flow", "overflow.port_a.m_flow",
- "tank3.ports[2].m_flow"},
+ y = {"upperTank.ports[1].m_flow", "pipe.port_a.m_flow", "overflow.port_a.m_flow"},
```
Comment (beutlich) [inline Modelica/Resources/Scripts/Dymola/Fluid/TanksWithOverflow/plot level and port.m_flow.mos:67]: Why not `lowerTank.ports[2].m_flow`?

Comment (HansOlsson) [reply Modelica/Resources/Scripts/Dymola/Fluid/TanksWithOverflow/plot level and port.m_flow.mos:67]: I experimented with that - but it's the same as `overflow.port_a.m_flow`; so it seemed redundant.

Comment (HansOlsson) [reply Modelica/Resources/Scripts/Dymola/Fluid/TanksWithOverflow/plot level and port.m_flow.mos:67]: And easier to understand what overflow is.

**Review by beutlich — APPROVED** [review]

**Review by m-kessler — APPROVED** [review]
Your changes are all fine.

However, the scripts are not very good and should have some general clean-up. But maybe this is another issue.

**Comment by beutlich** [comment]
> However, the scripts are not very good and should have some general clean-up. But maybe this is another issue.

@casells Perhaps, MAP-Lib could spend some of its budget to convert the vendor-specific plot scripts to specified figure annotations.

**Pull request merged**